lib/internal/modules/cjs/loader.js

```diff
@@ -172,7 +172,13 @@ const moduleParentCache = new SafeWeakMap();
 function Module(id = '', parent) {
   this.id = id;
   this.path = path.dirname(id);
```

Comment:
What about these two?

Reply:
They are "vulnerable" too, but I'm worried they might be people using it to monkey-patch the CJS internals. They can be handled in a follow-up PR that can be labelled as `do-not-land-on-lts`, this PR focuses on `package.json` fields only (for which I don't think of any actual use-case).

Reply:
Maybe patching exports/main/etc at runtime? but fair enough